tools/clang/lib/SPIRV/EmitVisitor.cpp

```diff
@@ -1026,5 +1091,401 @@ uint32_t EmitTypeHandler::emitType(const SpirvType *type,
   return id;
 }
 
+std::pair<uint32_t, uint32_t>
```

Comment:
Better to put a TOOD here to mention that this function should be merged into the Type class hierarchy

Reply:
Done